metricflow/sql/render/snowflake.py

```diff
@@ -0,0 +1,28 @@
+from metricflow.sql.render.expr_renderer import (
+    DefaultSqlExpressionRenderer,
+    SqlExpressionRenderer,
+    SqlExpressionRenderResult,
+)
+from metricflow.sql.render.sql_plan_renderer import DefaultSqlQueryPlanRenderer
+from metricflow.sql.sql_bind_parameters import SqlBindParameters
+from metricflow.sql.sql_exprs import SqlGenerateUuidExpression
+
+
+class SnowflakeSqlExpressionRenderer(DefaultSqlExpressionRenderer):
+    """Expression renderer for the Snowflake engine."""
+
+    def visit_generate_uuid_expr(self, node: SqlGenerateUuidExpression) -> SqlExpressionRenderResult:  # noqa: D
+        return SqlExpressionRenderResult(
+            sql="UUID_STRING()",
```

Comment:
Let us all pause for a moment to be irritated about how inconsistent this function naming is across engines...